### PR TITLE
Update modern-desktop-deployment-and-management-lab.md

### PR DESCRIPTION
**Pull request by cdmm12**
Updates to product versions and lab expiration dates

### CODE REVIEW

**Comment by cdmm12** [comment]
@bot agree

**Comment by cdmm12** [comment]
@bot agree

File: microsoft-365/enterprise/modern-desktop-deployment-and-management-lab.md [hunk 1]
```diff
@@ -20,23 +20,21 @@ description: Learn about where to access the Windows and Office Deployment Lab K
...
 
-The Windows and Office 365 deployment lab kit is designed to help you plan, test, and validate your deployment and management of desktops running Windows 10 Enterprise or Windows 11 Enterprise and Microsoft 365 Apps for enterprise. The labs in the kit cover using Microsoft Endpoint Configuration Manager, OneDrive, Windows Autopilot, and more. This kit is highly recommended for organizations preparing for desktop upgrades. As an isolated environment, the lab is also ideal for exploring deployment tool updates and testing your deployment-related automation.
-
-There are two versions of the lab available for free download:  
+The Windows and Office 365 deployment lab kits are designed to help you plan, test, and validate your deployment and management of desktops running Windows 10 Enterprise or Windows 11 Enterprise and Microsoft 365 Apps. The labs in the kit cover using Microsoft Intune and Microsoft Configuration Manager. This kit is highly recommended for organizations preparing for desktop upgrades. As an isolated environment, the lab is also ideal for exploring deployment tool updates and testing your deployment-related automation. There lab kits are for free download:
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/modern-desktop-deployment-and-management-lab.md]:
```suggestion
The Windows and Office 365 deployment lab kits are designed to help you plan, test, and validate your deployment and management of desktops running Windows 10 Enterprise or Windows 11 Enterprise and Microsoft 365 Apps. The labs in the kit cover using Microsoft Intune and Microsoft Configuration Manager. This kit is highly recommended for organizations preparing for desktop upgrades. As an isolated environment, the lab is also ideal for exploring deployment tool updates and testing your deployment-related automation. The following lab kits are available for free download:  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cdmm12, kindly commit the suggested change. Many thanks.

File: microsoft-365/enterprise/modern-desktop-deployment-and-management-lab.md (hunk 1 above)
Comment (cdmm12) [reply microsoft-365/enterprise/modern-desktop-deployment-and-management-lab.md]: Johan - can you let me know the steps required on my side to approve this change?

**Comment by cdmm12** [comment]
@bot agree

**Comment by cdmm12** [comment]
@microsoft-github-policy-service agree

File: microsoft-365/enterprise/modern-desktop-deployment-and-management-lab.md [hunk 2]
```diff
@@ -20,23 +20,21 @@ description: Learn about where to access the Windows and Office Deployment Lab K
...
 
 |Windows 10 Lab|Windows 11 Lab|
 |---|---|
-|[Windows 10 lab environment](https://download.microsoft.com/download/8/5/e/85e007b0-1f3e-460c-bd0a-5a8c6ec490b5/Win10_21H2_lab.zip)|[Windows 11 lab environment](https://download.microsoft.com/download/5/0/b/50bbe36a-9291-4339-9dcc-2a444fcd1659/Microsoft365DeviceLabKit.zip)|
-|[Windows 10 lab guides](https://download.microsoft.com/download/b/d/4/bd4f430b-8cd1-4a07-97b1-c32100fce7ae/Win_10_21H2_lab_guides.zip)|[Windows 11 lab guides](https://download.microsoft.com/download/5/0/b/50bbe36a-9291-4339-9dcc-2a444fcd1659/Win11_SetUp_Guide_08.05.zip)|
+|[Windows 10 lab environment](https://download.microsoft.com/download/b/7/6/b7696d5b-940e-4af6-ba8b-32cfa3532e6e/Windows10_21H2_2022-10-19.zip)|[Windows 11 lab environment](https://download.microsoft.com/download/a/1/0/a10d1f67-b499-4c2f-8db1-79d29cd98b05/Windows11_21H1_2022-10-18.zip)|
+|[Windows 10 lab guides](https://download.microsoft.com/download/b/7/6/b7696d5b-940e-4af6-ba8b-32cfa3532e6e/Win10_21H2_lab_guide_10.18.zip)|[Windows 11 lab guides](https://download.microsoft.com/download/a/1/0/a10d1f67-b499-4c2f-8db1-79d29cd98b05/Win11_Lab_Guides_10.18.zip)|
```
Comment (aczechowski) [inline microsoft-365/enterprise/modern-desktop-deployment-and-management-lab.md]: @cdmm12 the first link for *Windows 10 lab guides* doesn't work for me: `https://download.microsoft.com/download/b/7/6/b7696d5b-940e-4af6-ba8b-32cfa3532e6e/Win10_21H2_lab_guide_10.18.zip`
can you confirm?

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by aczechowski — CHANGES_REQUESTED** [review]
see comments inline re bad link

**Review by aczechowski — APPROVED** [review]

**Pull request merged**